Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```diff
@@ -391,6 +392,9 @@
       - WeaponLaserCannon
       - WeaponLaserCarbine
       - WeaponXrayCannon
+      - WeaponCrusher
```

Comment:
Making these researchable just- removes some progression again. Also, I guarantee, some salvs will refuse to do their jobs until these are researched.

Reply:
you can get them roundstart just by doing debris, its more for if you get really unlucky and cant get enough for the whole team. salv not doing their job has always been a player issue and qm should fire them, you dont need to wait 2 hours for a glaive just to kill debris fish